mindsdb_native/libs/data_sources/file_ds.py

```diff
@@ -203,4 +202,5 @@ def _getDataIo(self, file):
             # No file type identified
             return data, None, dialect
 
-   
\ No newline at end of file
+    def name(self):
+        return 'File, {}'.format(self.file)
```

Comment:
One small change needed here, only use the last node in self.file.

So, if self.file is `/home/george/hdi.csv` make it only `hdi.csv` ... different from what I originally  said but I just realized we'll have problems with exposing internal pathing in mindsdb server otherwise and this will confuse people.